index.html

```diff
@@ -73,6 +76,196 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Tonic</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 2 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio2.png" alt="Project 2" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Multi-Post Stories</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 3 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio3.png" alt="Project 3" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Tonic</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 4-->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/snapshoot-portfolio4.png" alt="Project 4" />
+          </div>
+          <div class="work_details">
+            <h1 class="work_title">Multi-Post Stories</h1>
+            <ul class="work_info">
+              <li class="w_s">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. The typo(font, color), and spacing(padding, margin) are supposed to be similar to the template in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198299962-1358fc24-3958-4a4d-8343-2d8e1ed904cc.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198300179-e96f0552-a508-4a00-9d3e-068315e3e585.png)

`HINT: You may consider using the flexbox or grid container to place these elements on the page`